Model/Sales/MagentoCollection.php

```diff
@@ -90,6 +91,7 @@ class MagentoCollection implements MagentoCollectionInterface
         'insurance'                         => null,
         'label_amount'                      => NewShipment::DEFAULT_LABEL_AMOUNT,
         'digital_stamp_weight'              => null,
+        'retour_in_the_box'                 => false,
```

Comment:
Noem het overal "return", "retour" is geen Engels.

Reply:
Heb dit gedaan